src/pages/GetInvolved/GetInvolved.js

```diff
@@ -1,12 +1,42 @@
-import React from 'react';
-import Layout from 'components/Layout/Layout';
+import React from "react";
+import { useSelector } from "react-redux"
+import Layout from "components/Layout/Layout";
+import { Link } from "react-router-dom";
+import './GetInvolved.scss';
 
 const GetInvolved = () => {
+
+  const clubDataAll = useSelector(state => state.clubs)
+
+  const renderClubs = clubs => {
+    return clubs.map((club) => {
+      return (
+        <li>
+          <Link className="get-involved__items--link" key={club.id} to={club.link}>
```

Comment:
- Follow BEM naming for classNames.
- This is a single item so className should be `item`.